.github/workflows/test.yml

```
@@ -0,0 +1,19 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
```

```suggestion
  pull_request:
    branches:
      - main
  workflow_dispatch:
```

to manually run the tests on any branch/commit from the Actions page